Actions/BuildPowerPlatform/action.yaml

```diff
@@ -71,7 +74,7 @@ runs:
         ${{ github.action_path }}\updateEnvironmentAndCompany.ps1 -companyId $ENV:_companyId -environmentName $ENV:_environmentName -solutionFolder $ENV:_solutionFolder
     
     - name: Test Pack (Not real - just workaround for https://github.com/microsoft/powerplatform-vscode/issues/412)
-      uses: microsoft/powerplatform-actions/pack-solution@v0
+      uses: microsoft/powerplatform-actions/pack-solution@v1
```

Comment:
@freddydk Didn't we merger this into the PPPreview branch in the last update?

Reply:
I cannot remember that, sorry